packages/store-ui/src/molecules/Slider/Slider.tsx

```diff
@@ -104,7 +106,7 @@ const Slider = ({
           setMinVal(value)
           minValRef.current = value
         }}
-        data-store-slider-thumb="left"
+        data-store-input="slider-thumb-left"
```

Comment:
I don't think we should change this. I think it's both a `data-store-input` and a `data-store-slider-thumb`. It could be `data-store-input={false}` and `data-store-slider-thumb` as well, I'm not quite sure what's the best approach here.

Reply:
Following the carousel implementation, we should use the `data-thumb="left|right"` (or data-slider-thumb)